### PR TITLE
[db] initial district implementation

### PR DESCRIPTION
**Pull request by Infinidrix**

### CODE REVIEW

File: Hack/Backend/controllers/DistrictController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+exports.getSymptomUsersByDistrict = async(req, res) => {
+    if (!req.param.district){
+        return res.status(400).send("District name not specified.");
+    }
+
+    try {
+        const districtResult = await DistrictModel.findOne({name: req.param.district});
+        if (districtResult){
+            const resultInDistrict = await User.find({district: districtResult._id});
```
Comment (Khalid-Sultan) [inline Hack/Backend/controllers/DistrictController.js:13]: I assume you wanted to search within LocationUser and not the actual User Model

Comment (Infinidrix) [reply Hack/Backend/controllers/DistrictController.js:13]: Yes yes you're right.

File: Hack/Backend/test/SymptomUserHistoryControllerTest.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ describe("Symptom User History API", () => {
   beforeEach(async () => {
     user = new User({
       _id: mongoose.Types.ObjectId(),
-      username: `${Date.now().toString()}`,
+      username: `${Date.now().toString()} ${Math.random()}`,
```
Comment (Khalid-Sultan) [inline Hack/Backend/test/SymptomUserHistoryControllerTest.js:24]: It would be nice if we can propagate this change for other test functions as well

Comment (Infinidrix) [reply Hack/Backend/test/SymptomUserHistoryControllerTest.js:24]: Sure thing

**Pull request merged**